src/tiff/libtiff/tif_config.h

```diff
@@ -364,8 +364,12 @@
 /* Set the native cpu bit order */
 #define HOST_FILLORDER FILLORDER_LSB2MSB
 
-/* MSVC 14 does have snprintf() and doesn't allow defining it */
-#if !defined(_MSC_VER) || _MSC_VER < 1900
+/* MSVC 14 does have snprintf() and doesn't allow defining it.  Also MinGW32
```

Comment:
Would it be possible to use exactly the same comment and `#if` syntax as in master? This one has the same effect, but it just doesn't seem right to have diverging fixes in the two branches, why not to use exactly the same one?

Reply:
Done and pushed.

I did this under windows and using Tortoisegit. It doesn't seem to like switch from master to WX_3_0_BRANCH... because of the submodules.